R/z_animintHelpers.R

```diff
@@ -762,7 +762,14 @@ getCommonChunk <- function(built, chunk.vars, aes.list){
   ## so that common.not.na is not empty
   ## due to the plot's alpha, stroke or other columns
   built <- as.data.table(built)
-  built <- built[,lapply(.SD, function(x) {if(all(is.na(x))) {NULL} else {x}} )]
+  ## convert 'built' from df to dt to improve speed
+  built <- built[, lapply(.SD, function(x) {
+    if (all(is.na(x))) {
+      NULL
+      } else {
+        x
+        }
```

Comment:
for closing parens, please use same indentation as the line which opened them (here you have 2 and 4 too many spaces)